pkg/sqlutil/monitor.go

```diff
@@ -0,0 +1,146 @@
+package sqlutil
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+)
+
+const slowMsg = "SLOW SQL QUERY"
+
+var promSQLQueryTime = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "sql_query_timeout_percent",
```

Comment:
I don't think we will be able to swap this in for everything at once in core, so we'll have to find a way to share this prom metric during the transition. Maybe temporarily exporting is the simplest thing to do :thinking: